.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "tests/ubench/google-benchmark"]
```

Comment:
We removed all of the submodules because they were overcomplicating things.

There are two approaches that I would prefer:
- include a copy of Google Benchmark in the source
- make it a requirement that the user must install.